TASVideos/WikiModules/PublicationsByPlatform.cshtml.cs

```diff
@@ -0,0 +1,50 @@
+﻿using TASVideos.WikiEngine;
+
+namespace TASVideos.WikiModules;
+
+[WikiModule(ModuleNames.PublicationsByPlatform)]
+public class PublicationsByPlatform(IGameSystemService platforms, IClassService classes) : WikiViewComponent
+{
+	public IReadOnlyList<(string DisplayName, string Code)> Platforms { get; private set; } = null!;
+
+	public IReadOnlyCollection<PublicationClass> PubClasses { get; set; } = null!;
+
+	public async Task<IViewComponentResult> InvokeAsync(IList<string> groupings)
+	{
+		var extant = (await platforms.GetAll()).ToList();
+		List<IReadOnlyList<SystemsResponse>> rows = [];
+		void ProcessGroup(string groupStr)
```

Comment:
Is this local function necessary here? It seems to be called only once, and it makes the entire method more difficult to parse and understand, because I have to keep going up and down to see what things are affected where.

Reply:
It's that or `goto`, since C# doesn't have multi-`break`.